lighthouse-core/lib/i18n/i18n.js

```diff
@@ -473,6 +473,17 @@ function replaceIcuMessageInstanceIds(inputObject, locale) {
   return icuMessagePaths;
 }
 
+/** @typedef {import('./locales').LhlMessages} LhlMessages */
+
+/**
+ * Populate the i18n string lookup dict with locale data
```

Comment:
should we link off to that design and/or mention how this is expected to be used?